moz_security/sandboxes/heka/analysis/moz_security_auth_lastx.lua

```diff
@@ -295,6 +306,16 @@ function process_message()
         payload = payload .. string.format("WARNING, unacceptable drift %d seconds\n", delaysec)
     end
 
+    if secm then
+        secm:reset()
```

Comment:
That is true for this particular case, but in cases where we summarize on timer_event (e.g., hh) it would be a larger aggregation. It seems simpler to just generate it in process_event like this since these are low volume vs. have a timer event / interval associated with generating it.